demos/common/samples/args_helper.hpp

```diff
@@ -34,8 +34,10 @@
 inline void readInputFilesArguments(std::vector<std::string> &files, const std::string& arg) {
     struct stat sb;
     if (stat(arg.c_str(), &sb) != 0) {
-        slog::warn << "File " << arg << " cannot be opened!" << slog::endl;
-        return;
+        if (arg.find("rtsp:") == std::string::npos) {
```

Comment:
A better test here would be `!= 0`, since something like `foo/rtsp:bar` isn't a camera address.

Reply:
Fixed

Reply:
Fixed.